### ISSUE TITLE
[Feature Request] Widget for Nextcloud

### ISSUE THREAD

**Issue by AtlanGon** [benphelps/homepage#257]
Could you make a widget for Nextcloud? thanks

Here are some infos.

url: https://<domain>/ocs/v2.php/apps/serverinfo/api/v1/info?format=json

relevant json data (for what I see it):
```
{
  "ocs": {
    "meta": {
      "message": "OK",
      "status": "ok",
      "statuscode": 200
    },
    "data": {
      "activeUsers": {
        "last5minutes": 1,
        "last24hours": 1,
        "last1hour": 1
      },
      "nextcloud": {
        "shares": {
          "num_fed_shares_received": 0,
          "num_fed_shares_sent": 0,
          "num_shares_groups": 2,
          "num_shares_link_no_password": 0,
          "num_shares_user": 0,
          "num_shares_room": 0,
          "num_shares_mail": 0,
          "num_shares_link": 0,
          "num_shares": 6,
          "permissions_2_31": "4",
          "permissions_1_31": "2"
        },
        "storage": {
          "num_storages_home": 13,
          "num_storages_local": 1,
          "num_files": 53845,
          "num_users": 12,
          "num_storages": 14,
          "num_storages_other": 0
        }
      }
    }
  }
}
```

But I don't know how you would tell the widget what to display.
Maybe like this: (but that would mean a lot of work for you :weary: )

```
widget:
  type: nextcloud
  url: http://...
  1: meta.status
  2: data.activeUsers.last1hour
  3: data.nextcloud.shares.num_shares
```

three or four should be enough.

**Comment by galgofa** [comment]
+1

**Comment by Samurai1201** [comment]
+1

**Comment by Mbarmem** [comment]
Please let me know your opinions on this.

![image](https://user-images.githubusercontent.com/52536342/219314162-3b212eae-d1e4-4c1d-a980-bdb3696503ca.png)

**Comment by shamoon** [comment]
Closed by #1010
